typhos/utils.py

```diff
@@ -102,12 +103,23 @@ def grab_kind(device, kind):
                                 device.configuration_attrs]}[kind]
     # Return that kind filtered for devices
     signals = collections.OrderedDict()
-    device_class = device.__class__
     for attr in kind_attr:
-        signal = getattr(device, attr)
+        if '.' in attr:
+            hierarchy = attr.split('.')
```

Comment:
This is a big regression in ophyd if this isn't working - I wonder why... ?

https://github.com/bluesky/ophyd/blob/0c9883548db66b334892140f5875405a5b72a419/ophyd/device.py#L1184